handsontable/.config/test-e2e-cjs.js

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
The file's content is almost the same as the `test-e2e.js` file. It can be modified by this:
```js
const configFactory = require('./test-e2e');

module.exports.create = function create(envArgs) {
  const config = configFactory.create(envArgs);

  config.forEach(function(c) {
    c.resolve.alias.handsontable = path.resolve(__dirname, '../dist/handsontable-cjs.js');
  });

  return [].concat(config);
}
```
Can you check that?